react-native-libraries.json

```diff
@@ -5088,5 +5089,16 @@
     "npmPkg": "rn-tourguide",
     "unmaintained": false,
     "examples": ["https://github.com/xcarpentier/rn-tourguide/blob/master/App.tsx"]
+  },
+  {
+    "githubUrl": "https://github.com/react-native-community/react-native-slider",
+    "npmPkg": "@react-native-community/slider",
+    "images": [
+      "https://i.postimg.cc/dQTYzGD5/Screenshot-2019-03-25-at-11-24-59.png", 
+      "https://i.postimg.cc/CKdtbVqc/Screenshot-2019-03-25-at-11-26-54.png"
+    ],
+    "ios": true,
+    "android": true,
```

Comment:
with sdk 38 we will support it in expo managed workflow as well, i'll update this entry after we release :)